src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.subway.line.domain;
+
+import static java.util.stream.Collectors.*;
+import static javax.persistence.CascadeType.*;
+import static javax.persistence.FetchType.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+
+@Embeddable
+public class Sections {
+
+    @OneToMany(mappedBy = "line", fetch = LAZY, cascade = ALL)
+    private List<Section> sections = new ArrayList<>();
+
+    public void add(List<Section> sections, Line line) {
+        this.sections = sections.stream()
+                                .peek(section -> section.setLine(line))
+                                .collect(toList());
+    }
+
+    public List<Section> getStations() {
+        List<Section> sortSections = sections.stream().sorted().collect(toList());
```

Comment:
하나의 라인에 하나의 점을 사용하면 가독성이 좋아집니다 :)
```suggestion
        List<Section> sortSections = sections.stream()
                .sorted()
                .collect(toList());
```